.github/workflows/validate_linux_rt.yml

```diff
@@ -22,23 +22,23 @@ jobs:
     - name: Get the latest tag
       id: latesttag
       run: >-
-        echo "::set-output name=tag::$(
+        echo "tag=$(
           git tag --list --sort=-version:refname | grep '^\(v[0-9]\+.[0-9]\+.[0-9]\+\)$' | head -1
-        )"
+        )" >> $GITHUB_OUTPUT
 
     - name: Get latest tag without leading v
       id: latesttagwithoutv
       run: >-
-        echo "::set-output name=tag::$(
+        echo ":tag=$(
```

Comment:
Left a leading `:` in front of the `tag=` here.

Reply:
Fixed in latest commit